.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,64 @@
+name: code-coverage
```

Comment:
"Code Coverage" instead (just to be consistent with the naming of the other workflows)